nautobot/extras/forms/forms.py

```diff
@@ -1891,11 +1925,8 @@ class RoleBulkEditForm(NautobotBulkEditForm):
     color = forms.CharField(max_length=6, required=False, widget=ColorSelect())
     description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
     weight = forms.IntegerField(required=False)
-    add_content_types = MultipleContentTypeField(
-        queryset=RoleModelsQuery().as_queryset(), required=False, label="Add Content Type(s)"
-    )
-    remove_content_types = MultipleContentTypeField(
-        queryset=RoleModelsQuery().as_queryset(), required=False, label="Remove Content Type(s)"
+    content_types = MultipleContentTypeField(
+        queryset=RoleModelsQuery().as_queryset(), required=False, label="Content Type(s)"
     )
```

Comment:
Why did we change this and these fields in `StatusBulkEditForm`? Please revert.

Reply:
This still needs to be reverted.